tests/test_job.py

```diff
@@ -45,6 +45,25 @@ def test_job_submit(hq_env: HqEnv):
     assert table[3][:3] == ["3", "sleep", "FINISHED"]
 
 
+def test_custom_name(hq_env: HqEnv, tmp_path):
+    hq_env.start_server()
+
+    hq_env.command(["submit", "sleep", "1", "--name=sleep_prog"])
+    time.sleep(0.2)
+    table = hq_env.command("jobs", as_table=True)
+    assert len(table) == 2
+    assert table[1][:3] == ["1", "sleep_prog", "WAITING"]
+    failed = False
+    try:
+        hq_env.command(["submit", "sleep", "1", "--name=second_sleep \t"])
+        hq_env.command(["submit", "sleep", "1", "--name=second_sleep \n"])
```

Comment:
This should be put into another `try`/`except` block, otherwise the second command won't ever be executed.

Reply:
I was waiting for a comment, I didn't like how having two try blocks looked 🙃